### PR TITLE
Simple Calculator using JavaScript

### PR DESCRIPTION
**Pull request by krishprav**
## **Add the working demo (screen recording) below _(Required)_**
271848316-8e01c7e4-db58-4226-b4c3-87918365b487

## Describe your changes
A Simple Calculator using Js

## Issue ticket number and link
#5

### CODE REVIEW

**Comment by anishaswain** [comment]
Please follow the folder structure https://github.com/anishaswain/Simple-JS-Projects/pull/14/files (i.e Project2/your_git_username/your_solution)
- project2 -> krishprav -> your html/css/js files

**Review by anishaswain — CHANGES_REQUESTED** [review]
Please remove files outside project2 folder

**Comment by anishaswain** [comment]
@krishprav your commits have deleted a few default files of the repo like `readme.md` . Please revert your commit

**Review by anishaswain — CHANGES_REQUESTED** [review]
Revert the deleted files

**Comment by anishaswain** [comment]
@krishprav please revert any changes expect the changes to your project2 folder

**Comment by krishprav** [comment]
sorry this my first contribution so i got confused now i am getting familiar.

File: README.md
Comment (anishaswain) [inline README.md:1]: revert this file change

File: pull_request_template.md
Comment (anishaswain) [inline pull_request_template.md:1]: Please revert this file changes

**Review by anishaswain — CHANGES_REQUESTED** [review]

**Pull request merged**